server/src/settings.ts

```diff
@@ -43,7 +45,15 @@ export class Settings {
   }
 
   get controllersPath() {
-    return `${this.projectPath}/app/javascript/controllers`
+    return `${this.projectPath}/${this.controllersRelativePath}`
+  }
+
+  get controllersRelativePath() {
+    if (existsSync(path.join(this.projectPath, "artisan"))) {
+      return "resources/js/controllers"
+    }
```

Comment:
I don't know if this even works. I'll test it more when/if [the ](https://github.com/marcoroth/stimulus-parser/pull/14) gets merged.

Reply:
This shouldn't be needed anymore, the Stimulus Parser now automatically detects and provides the root folders for controllers